src/ui/shared/round/OnlineRound.ts

```diff
@@ -185,6 +188,14 @@ export default class OnlineRound implements OnlineRoundInterface {
   public showActions = () => {
     router.backbutton.stack.push(this.hideActions)
     this.vm.showingActions = true
+
+    const d = this.data
+    if (!d || !d.player.user || !d.opponent.user)
+      return
+    xhr.getCrosstable(d.player.user.id, d.opponent.user.id).then(s => {
```

Comment:
It is unnecessary to request the crosstable each time the popup is opened. You should do it only once.

Reply:
I thought it was worth requesting each time in case the game ended and the score changed, but in retrospect the cost/benefit of that decision doesn't make any sense. Fixed.

Reply:
Then you could add a method `updateCrosstable()` and call it first time before showing popup, and once when the game has just ended (there is already a hook for that).